32.mender-convert/docs.md

```diff
@@ -8,6 +8,54 @@ github: false
 ---
 
 
+## mender-convert 4.0.3
+
+_Released 10.18.2023_
+
+### Statistics
+
+| Developers with the most changesets | |
+|---|---|
+| Kristian Amlie | 1 (33.3%) |
+| Michael Mugnai | 1 (33.3%) |
+| Lorenzo Ruffati | 1 (33.3%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Michael Mugnai | 19 (76.0%) |
+| Lorenzo Ruffati | 5 (20.0%) |
+| Kristian Amlie | 1 (4.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 1 (33.3%) |
+| michael.mugnai@gmail.com | 1 (33.3%) |
+| lorenzo.ruffati@unikie.com | 1 (33.3%) |
+
+| Top lines changed by employer | |
+|---|---|
+| michael.mugnai@gmail.com | 19 (76.0%) |
+| lorenzo.ruffati@unikie.com | 5 (20.0%) |
+| Northern.tech | 1 (4.0%) |
+
+| Employers with the most hackers (total 3) | |
+|---|---|
+| michael.mugnai@gmail.com | 1 (33.3%) |
+| lorenzo.ruffati@unikie.com | 1 (33.3%) |
```

Comment:
unikie.com should be added to the [company map](https://github.com/mendersoftware/integration/blob/master/extra/gitdm/company-map) before rendering.

Reply:
fatal.se too, if there is a company associated with it.

Reply:
I cannot find a company name for fatal.se